x-pack/filebeat/module/salesforce/logout/config/logout.yml

```diff
@@ -0,0 +1,53 @@
+{{ if .httpjson.enabled }}
+
+type: httpjson
+
+interval: {{ .interval }}
+request.method: GET
+auth.oauth2:
+  enabled: true
+  client.id: {{ .clientid }}
+  client.secret: {{ .clientsecret}}
+  token_url: https://login.salesforce.com/services/oauth2/token

```

Comment:
My knowledge might be outdated, but Salesforce still hasn't provided any self-hosted options, correct? Is it always https://login.salesforce.com/?

Reply:
Yes, that's correct. And yes, it's always https://login.salesforce.com/ ([reference](https://developer.salesforce.com/docs/atlas.en-us.api_iot.meta/api_iot/qs_auth_access_token.htm)). Anyways, let's add `token_url` as a user-configurable variable with the default value as `https://login.salesforce.com/services/oauth2/token`. Does that make sense?

Reply:
Yes, it makes sense. Thanks for clarifying it!

If you want to mock API in integration tests, it would be a good option to use.